File: leetcode/arrays/01-two-sum/my_solution.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        for i in range(0, len(nums)):
            for j in range(i+1, len(nums)):
                if (nums[i]+nums[j]==target):
                    return [i,j]

# Runtime: 4895 ms, faster than 10.48% of Python3 online submissions for Two Sum.
# Memory Usage: 14.8 MB, less than 92.24% of Python3 online submissions for Two Sum.

File: leetcode/arrays/88-merge-sorted-array/my-solution.java
class Solution {
    public void merge(int[] nums1, int m, int[] nums2, int n) {
        
        int[] tempArray = new int[m+n];
        int leftPointer = 0;
        int rightPointer = 0;
        int tempPointer = 0;
        
        while(leftPointer < m && rightPointer < nums2.length) {
            if(nums1[leftPointer] <= nums2[rightPointer]) {
                tempArray[tempPointer] = nums1[leftPointer];
                leftPointer++;
            } else {
                tempArray[tempPointer] = nums2[rightPointer];
                rightPointer++;
            }
            tempPointer++;
        }
        
        if(leftPointer < m) {
            while(leftPointer < m) {
                tempArray[tempPointer] = nums1[leftPointer];
                leftPointer++;
                tempPointer++;
            }
        } else if(rightPointer < nums2.length) {
            while(rightPointer < nums2.length) {
                tempArray[tempPointer] = nums2[rightPointer];
                rightPointer++;
                tempPointer++;
            }
        }
        
        for(int i=0; i<tempArray.length; i++) {
            nums1[i] = tempArray[i];
        }
    }
}

// Runtime: 1 ms, faster than 22.36% of Java online submissions for Merge Sorted Array.
// Memory Usage: 40 MB, less than 6.00% of Java online submissions for Merge Sorted Array.